libs/validate_results.py

```diff
@@ -71,10 +72,15 @@ def validate_counties_df(key, counties_df):
     _raise_error_if_incorrect_headers(key, RESULT_DATA_COLUMNS_COUNTIES, counties_df)
 
     # assert data from each of the states
+    expected_missing = EXPECTED_MISSING_STATES.union(
+        EXPECTED_MISSING_STATES_FROM_COUNTES
+    )
+    if key == "counties.OBSERVED_INTERVENTION":
```

Comment:
I'm not 100% sure where this is called from, but it would be better to pass in the actual enum here.

After looking, it looks like it would be fairly straightforward to pass it in.  Then you could just do:
```if intervention is Intevention.OBSERVED_INTERVENTION:```
Future proofs us for if (when) we end up changing the intervention name

Reply:
Good suggestion. Done, thanks!